src/app/lib/config/services/amharic.js

```diff
@@ -158,6 +158,7 @@ export const service = {
         video: 'ቪዲዮ',
         listen: 'ያድምጡ',
         watch: 'ተመልከት',
+        listenLive: 'በቀጥታ ያድምጡ ',
```

Comment:
There is an extra space after the words here but maybe it doesn't matter

Reply:
Thanks! Missed that one